recipe/migrations/fftw-3.3.8.yaml

```diff
@@ -0,0 +1,20 @@
+# To learn more about migrations read CFEP-09
+# https://github.com/conda-forge/conda-forge-enhancement-proposals/blob/master/cfep-09.md
+# The timestamp of when the migration was made
+# Can be obtained by copying the output of
+# python -c "import time; print(f'{time.time():.0f}')"
+migrator_ts: 1571566507
+__migrator:
+  kind:
+    version
+  # Only change the migration_number if the bot messes up,
+  # changing this will cause a complete rerun of the migration
+  migration_number:
+    1
+
+pin_run_as_build:
+  fftw:
+    max_pin: x.x
```

Comment:
Looking at https://abi-laboratory.pro/index.php?view=timeline&l=fftw this should be `x`